setup.cfg

```diff
@@ -15,4 +15,4 @@ builtins =
 
 # ignore "Black would make changes" produced by flake8-black
 # see also https://github.com/houndci/hound/issues/1769
-ignore = BLK100
+ignore = BLK100,E121,E123,E126,E226,E24,E704,W503,W504
```

Comment:
That's a lot ignored. What are those and why are they ignored? I'd rather not do that.

Reply:
This is outdated now, I was manually adding the default ignores of flake8. It should be
```
extend-ignore = BLK100
```